gg.minion.Minion.metainfo.xml

```diff
@@ -0,0 +1,65 @@
+    <p>NOTE: Snapshots of settings and addons don't work with Linux paths.</p>
+    <p>NOTE: Hardware acceleration is disabled by default.</p>
+    <p>NOTE: Minion may not list AddOns for the first time you use it, restarting it should fix the issue.</p>
+  </description>
+  <releases>
+    <release version="3.0.11" date="2022-11-30"/>
+  </releases>
+  <screenshots>
+    <screenshot type="default">
+      <caption>The installed screen</caption>
+      <image type="source">https://raw.githubusercontent.com/zastrixarundell/flathub/gg.minion.Minion/screenshots/installed.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>The find screen</caption>
+      <image type="source">https://raw.githubusercontent.com/zastrixarundell/flathub/gg.minion.Minion/screenshots/find.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>The search menu</caption>
+      <image type="source">https://raw.githubusercontent.com/zastrixarundell/flathub/gg.minion.Minion/screenshots/search.png</image>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
```

Comment:
Attributes set to `none` can be dropped.

Reply:
Having all contents set to none, as Minion is just an addon manager, can I drop `content_rating` attribute altogether, or just keep i empty?